resources/views/search-friends.blade.php

```diff
@@ -1,7 +1,6 @@
 @extends('layouts.app')
 @section('content')
-
```

Comment:
the side bar is included on that page, that was one of the changes I made
you should test out the branch